CommandsTest.java

```diff
@@ -0,0 +1,38 @@
+
+import static org.junit.Assert.*;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * The test class CommandsTest.
+ *
+ * @author  Milena vallejos  * @version 18 de mayo 2016
+ */
+public class CommandsTest
+{
+    /**
+     * Default constructor for test class CommandsTest
+     */
+    @Test
+    public void testCommandCopy()
+    
+    
+    {
+        
+         
+     
+         assertTrue( Copy.copyFile("C:\\Users\\Milena\\Music\\Grapes - I dunno.mp3","C:\\Users\\Milena\\Documents"));
```

Comment:
The unit tests should run on any environment, you cannot create tests that depend on your local machine configuration